yarn.lock

```diff
@@ -0,0 +1,10294 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We should not have the yarn.lock file here as we are telling the candidate to use npm not yarn